Jenkinsfile-datastax

```diff
@@ -19,22 +19,15 @@
  */
 
 def initializeEnvironment() {
-  env.DRIVER_DISPLAY_NAME = 'CassandraⓇ Java Driver'
+  env.DRIVER_DISPLAY_NAME = 'Java Driver for Apache CassandraⓇ'
   env.DRIVER_METRIC_TYPE = 'oss'
-  if (env.GIT_URL.contains('riptano/java-driver')) {
-    env.DRIVER_DISPLAY_NAME = 'private ' + env.DRIVER_DISPLAY_NAME
-    env.DRIVER_METRIC_TYPE = 'oss-private'
-  } else if (env.GIT_URL.contains('java-dse-driver')) {
-    env.DRIVER_DISPLAY_NAME = 'DSE Java Driver'
-    env.DRIVER_METRIC_TYPE = 'dse'
-  }
 
   env.GIT_SHA = "${env.GIT_COMMIT.take(7)}"
   env.GITHUB_PROJECT_URL = "https://${GIT_URL.replaceFirst(/(git@|http:\/\/|https:\/\/)/, '').replace(':', '/').replace('.git', '')}"
   env.GITHUB_BRANCH_URL = "${GITHUB_PROJECT_URL}/tree/${env.BRANCH_NAME}"
   env.GITHUB_COMMIT_URL = "${GITHUB_PROJECT_URL}/commit/${env.GIT_COMMIT}"
 
-  env.MAVEN_HOME = "${env.HOME}/.mvn/apache-maven-3.3.9"
+  env.MAVEN_HOME = "${env.HOME}/.mvn/apache-maven-3.6.3"
```

Comment:
Drive-by fix; we bumped Maven versions on this runner some time ago but apparently didn't bump the version used in the Java builds